docs/source/introduction_guide.rst

```diff
@@ -193,8 +193,7 @@ For fast research prototyping, it might be easier to link the model with the dat
                                           transforms.Normalize((0.1307,), (0.3081,))])
             # data
             mnist_train = MNIST(os.getcwd(), train=True, download=True)
```

Comment:
```suggestion
            mnist_train = MNIST(os.getcwd(), train=True, download=True, transform=transform)
```

Reply:
I also found another place where there is a similar problem. I have fixed both in the 2nd commit.